rio_tiler/io/cogeo.py

```diff
@@ -685,3 +704,41 @@ def __attrs_post_init__(self):
             )
         )
         super().__attrs_post_init__()
+
+
+@attr.s
+class LocalTileMatrixSet:
+    """Fake TMS for local image."""
+
+    width: int = attr.ib()
+    height: int = attr.ib()
+    tile_size: int = attr.ib(default=256)
+    rasterio_crs: CRS = attr.ib(default=None)
```

Comment:
```suggestion
```

Reply:
@geospatial-jeff we need `rasterio_crs` attribute to keep the compatibility with other TMS. https://github.com/cogeotiff/rio-tiler/blob/master/rio_tiler/io/cogeo.py#L361

It should always be `None` to match the image dataset CRS (also None)